src/Platform.Eda.Cli/Commands/ConfigureEda/JsonProcessor/PutSubscriberProcessChain.cs

```
@@ -183,7 +183,11 @@ public async Task<int> ProcessAsync(string inputFolderPath, string env, Dictiona
                 }
 
                 // Step 4 - Create PutSubscriberFile object for further processing
-                WriteNormalWithFileName(fileRelativePath);
+                if (!noDryRun) // output File Ok only on dry run
```

So this will only be an 'OK' output for the validation, warning/error are not handled by this line?

---

No, those are separate.